source/versions/filenames.rst

```diff
@@ -0,0 +1,117 @@
+=============================
+Reading the Download Filename
+=============================
+
+When you download files, the name of the file will provide some important version information. The following sections 
+describe the information provided.
+
+.. _sponge-forge-file-name:
+
+SpongeForge
+===========
+
+.. note::
+
+    Our versioning policy was updated in October 2018. **Beginning with** SpongeAPI 8.0, the following information 
+    applies. See :doc:`legacy-versions` for information on older versions.
+
+When you download SpongeForge, the name of the file includes a Forge build number which this version of SpongeForge is 
+built with and guaranteed to be compatible. We do not support newer versions of Forge; however, we try to get an update 
+out when a Forge release breaks SpongeForge. You can try the Forge version needed as it will work fine most of the 
```

Comment:
This isn't true either.
SpongeForge is built against the latest **recommended** version of Forge, we don't bump the Forge version unless there's a new recommended build. 

See https://github.com/SpongePowered/SpongeForge/issues/2416 https://github.com/SpongePowered/SpongeForge/issues/2370

Reply:
Done